Vagrantfile

```diff
@@ -18,9 +18,9 @@ Vagrant.configure("2") do |config|
 
     # Run bootstrap.sh script on first boot:
     config.vm.provision "bootstrap", type: "shell", path: "bootstrap.sh"
-    config.vm.synced_folder  ".", "/vagrant",
+    config.vm.synced_folder  ".", "/vagrant", type: "rsync",
                              rsync__args: ["--verbose", "--archive", "--delete", "-z", "--links"]
-    config.vm.synced_folder "#{NTECH_ROOT}", "/northern.tech",
+    config.vm.synced_folder "#{NTECH_ROOT}", "/northern.tech/cfengine", type: "rsync",
```

Comment:
remove `/cfengine´.

Reply:
Oh, these changes shouldn't be in the commit at all. Good catch!